http-add-on/templates/deployment.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    control-plane: controller-manager
+    httpscaledobjects.http.keda.sh/version: {{ .Values.version }}
+    keda.sh/addon: keda-http-addon
+  name: keda-http-addon-controller-manager
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      control-plane: controller-manager
+      httpscaledobjects.http.keda.sh/version: {{ .Values.version }}
+      keda.sh/addon: keda-http-addon
+  template:
+    metadata:
+      labels:
+        control-plane: controller-manager
+        httpscaledobjects.http.keda.sh/version: {{ .Values.version }}
+        keda.sh/addon: keda-http-addon
+    spec:
+      containers:
+      - args:
+        - --secure-listen-address=0.0.0.0:8443
+        - --upstream=http://127.0.0.1:8080/
+        - --logtostderr=true
+        - --v=10
+        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.5.0
+        name: kube-rbac-proxy
+      - args:
+        - --metrics-addr=127.0.0.1:8080
+        - --enable-leader-election
+        image: {{ .Values.images.operator }}
+        imagePullPolicy: Always
+        name: manager
+        env:
+        - name: KEDAHTTP_OPERATOR_EXTERNAL_SCALER_IMAGE
+          value: {{ .Values.images.scaler }}
+        - name: KEDAHTTP_OPERATOR_INTERCEPTOR_IMAGE
+          value: {{ .Values.images.interceptor }}
+        # resources:
```

Comment:
Let's open this up through the values, but a dedicated issue to track this is OK.

Reply:
What do you mean? To split those keys?

Reply:
The resources are commented out in this section, but people should be able to configure them.

For example: https://github.com/kedacore/charts/blob/master/keda/templates/12-keda-deployment.yaml#L100-L101

Reply:
done